src/lifecycle/properties-created.js

```diff
@@ -17,6 +17,6 @@ export default function propertiesApply (elem, properties) {
     // Once that bug is fixed, the initial value being passed as the second
     // argument to prop.created() can use the overridden property definition to
     // get the initial value.
-    prop.created && prop.created(elem, initialValue);
+    prop.created(elem, initialValue);
```

Comment:
It did because we were possibly thinking about allowing a function to return a user-defined prop definition, but we haven't documented this and don't publicly support it yet. So currently, there will always be a `created` function here.